crates/e2e/src/setup/services.rs

```diff
@@ -39,6 +39,10 @@ pub fn orders_for_tx_endpoint(tx_hash: &H256) -> String {
     format!("/api/v1/transactions/{tx_hash:?}/orders")
 }
 
+pub fn orders_for_owner(owner: &H160, offset: u64, limit: u64) -> String {
```

Comment:
This and the 2 functions above can be private. Could even be inlined since they are only used at one place.
```suggestion 
fn orders_for_owner(owner: &H160, offset: u64, limit: u64) -> String {
```